### PR TITLE
#TP-12 ft:Reset password

### PR DESCRIPTION
**Pull request by jeandamascyifuzo**
### What this PR does
This Pr allows the driver or operator to reset their password.

 
- [x] **driver** and **operator** are able to provide their email.
- [x] Email containing a reset password link is sent to their email
- [x] **driver** and **operator** will be able to reset their password.
- [x] Appropriate error messages are displayed on unsuccessful attempts

Trello card Id

> #TP-12

 ### How you can manually test it

1. Clone the repo
2. checkout to ft-resetPass-integration-TP-12 branch
3. git pull origin ft-resetPass-integration-TP-12
4.  add this URL: BASE_URL=http://localhost:8080/newpassword/accounts/reset-password/   .env=>backend

**Install all dependencies using** 
`npm i`

**run development server using**
`npm run dev`

### TrelloCard

[link](https://trello.com/c/rgnoEv0m/12-reset-password)

**ScreenShot**
 ### unsuccessful attempts
Screen Shot 2022-04-28 at 16 37 57

 ### successful attempts
Screen Shot 2022-04-28 at 16 38 25

### CODE REVIEW

**Review by chanceDira — DISMISSED** [review]
**This really works fine 👍**

**Review by renepromesse — APPROVED** [review]

File: src/components/passwordReset/passwordereset.js [hunk 1]
```diff
@@ -5,135 +5,159 @@ import leftArrow from "../../assets/Image/left-arrow.svg";
...
+                                    <p className=" text-primary-500 font-extrabold text-2xl text-center mb-2">
+                                        Phantom
+                                    </p>
+                                    <p className=" text-white text-l font-bold text-center mb-5">
+                                        Forgot your password ?
+                                    </p>
+                                    <div className="flex justify-center items-center">
+                                        <form action="" onSubmit={formik.handleSubmit}>
+                                            <input
+                                                className=" flex justify-center w-60 bg-sky-50 w-full appearance-none rounded  py-2 px-3 mb-3 text-xs font-semibold text-gray-900 leading-tight "
```
Comment (sezeranoJchrisostome) [inline src/components/passwordReset/passwordereset.js:74]: Can you reset this input and disable it when it is still in progress

**Review by sezeranoJchrisostome — CHANGES_REQUESTED** [review]
Well done, @jeandamascyifuzo

File: src/components/passwordReset/passwordereset.js (hunk 1 above)
Comment (sezeranoJchrisostome) [reply src/components/passwordReset/passwordereset.js:74]: and also add notify

File: src/components/passwordReset/passwordereset.js [hunk 2]
```diff
@@ -5,135 +5,159 @@ import leftArrow from "../../assets/Image/left-arrow.svg";
...
+    });
+    const formik = useFormik({
+        initialValues: {
+            email: "",
+        },
+        validationSchema: Yup.object({
+            email: Yup.string().email("Invalid email").required("Required"),
+        }),
+        onSubmit: (values) => {
+            const { email } = values;
```
Comment (sezeranoJchrisostome) [inline src/components/passwordReset/passwordereset.js:34]: Can you call notify() here to show me that my request is being progressed?

**Review by sezeranoJchrisostome — CHANGES_REQUESTED** [review]

**Review by sezeranoJchrisostome — APPROVED** [review]

**Pull request merged**